storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst

```diff
@@ -119,6 +119,12 @@ backed up:
   to look up the ``innodb_to_lsn`` values in the
   :ref:`PERCONA_SCHEMA.xtrabackup_history <xtrabackup_history>` table.
 
+Using the keyring_file component requires the following privilege:
+
+.. sourcecode::
+
+   GRANT SELECT ON performance_schema.keyring_component_status TO bkpuser@'localhost' 
+
```

Comment:
move to `An SQL example of creating a database user with the minimum privileges required
to full backups would be:`

Reply:
OK.